misc/python/materialize/cli/ci_annotate_errors.py

```diff
@@ -521,5 +519,15 @@ def get_job_state() -> str:
     raise ValueError("Job not found")
 
 
+def get_suite_name() -> str:
+    suite_name = os.getenv("BUILDKITE_LABEL", "Unknown Test")
```

Comment:
(I wanted to suggest to fall back to the value of `BUILDKITE_STEP_KEY` instead but have just seen that that might also be empty.)